src/components/Modals/Settings/FiatCurrencies.tsx

```diff
@@ -0,0 +1,86 @@
+import { ArrowBackIcon } from '@chakra-ui/icons'
+import { Button, Flex, Icon, IconButton, ModalBody, ModalHeader } from '@chakra-ui/react'
+import { SupportedFiatCurrencies, SupportedFiatCurrenciesList } from '@shapeshiftoss/market-service'
+import { identity } from 'lodash'
+import sortBy from 'lodash/sortBy'
```

Comment:
smallest of comments, not exactly sure about tree-shaking with the explicit path imports vs destructed on top level lodash, but feel like either or would be better

Reply:
Yeah `lodash/functionName` should always be preferred unless you have some AST magic to automate that e.g babel plugin https://lodash.com/per-method-packages